broker/l2tp_broker.py

```diff
@@ -768,7 +768,15 @@ def hook(self, name, *args):
     # Execute the registered hook
     logger.debug("Executing hook '%s' via script '%s %s'." % (name, script, str([str(x) for x in args])))
     try:
-      gevent_subprocess.call([script] + [str(x) for x in args])
+      script_process = gevent.subprocess.Popen([script] + [str(x) for x in args], stdout=gevent.subprocess.PIPE, stderr=gevent.subprocess.PIPE)
+      stdout, stderr = script_process.communicate()
+
+      if stdout:
+          logger.debug("script stdout:")
```

Comment:
Can you instead use something like `"Hook '%s' script stdout:"`? Since `communicate` may yield, the output could appear after some other hooks have been executed and then you can't know which script produced the output.

Similar for stderr.